tests/python/gaia-ui-tests/gaiatest/apps/calendar/app.py

```diff
@@ -15,6 +15,7 @@
 class Calendar(Base):
 
     name = 'Calendar'
+    origin = 'app://calendar.gaiamobile.org'
```

Comment:
I think we can factorize the generation of the origin, as it's often the same pattern (`app://appname.gaiamobile.org`). In base.py:
``` python
    @property
    def name(self):
        raise NotImplementedError("{} should implement the class attribute called 'name'".format(self.__class__)  )

    @property
    def origin(self):
        return 'app://{}.gaiamobile.org'.format(self.name.lower())
```

And then we don't have to defined `origin`, except in Dialer and Contacts.